### ISSUE TITLE
If Authors == undefined throws error.

### ISSUE THREAD

**Issue by lbertenasco** [ngrx/example-app#8]
When searching for a book and a result has author == undefined
```javascript
vendor.bundle.js:3923 Uncaught EXCEPTION: Error in ./BookPreviewComponent class BookPreviewComponent - inline template:13:16
ORIGINAL EXCEPTION: TypeError: Cannot read property 'length' of undefined
ORIGINAL STACKTRACE:
TypeError: Cannot read property 'length' of undefined
    at AddCommasPipe.transform (http://localhost:8080/1.chunk.js:177:25)
    at http://localhost:8080/vendor.bundle.js:15211:23
    at DebugAppView._View_BookPreviewComponent0.detectChangesInternal (BookPreviewComponent.template.js:237:152)
    at DebugAppView.AppView.detectChanges (http://localhost:8080/vendor.bundle.js:44997:15)
    at DebugAppView.detectChanges (http://localhost:8080/vendor.bundle.js:45086:45)
    at DebugAppView.AppView.detectViewChildrenChanges (http://localhost:8080/vendor.bundle.js:45017:35)
    at DebugAppView._View_BookPreviewListComponent1.detectChangesInternal (BookPreviewListComponent.template.js:91:8)
    at DebugAppView.AppView.detectChanges (http://localhost:8080/vendor.bundle.js:44997:15)
    at DebugAppView.detectChanges (http://localhost:8080/vendor.bundle.js:45086:45)
    at DebugAppView.AppView.detectContentChildrenChanges (http://localhost:8080/vendor.bundle.js:45012:38)
ERROR CONTEXT:
[object Object]
```

**Comment by mikekidder** [comment]
I just submitted PR #12 to fix; minor change

**Comment by MikeRyan52** [comment]
Closed in #9
